index.html

```diff
@@ -0,0 +1,195 @@
+                    </button>
+                </div>
+            </div>
+            <div class="project-items">
+                <img src="images.svg\Rectangle 21 (19).svg" class="project-image" alt="">
+                <div class="languages">
+                    <h2>Project name goes here</h2>
+                    <ul class="H-R-J">
+                        <li class="Html-css">HTML/CSS</li>
+                        <li class="Ruby">RUBY ON RAILS</li>
+                        <li class="Javascript">JAVASCRIPT</li>
+                    </ul>
+                    <button class="see-this-project">
+                        See this project
+                        <img src="images.svg\Union (7).svg" alt="" class="arrow">
+                    </button>
+                </div>
+            </div>
+            <div class="project-items">
+                <img src="images.svg\Rectangle 21 (19).svg" class="project-image" alt="">
+                <div class="languages">
+                    <h2>Project name goes here</h2>
+                    <ul class="H-R-J">
+                        <li class="Html-css">HTML/CSS</li>
+                        <li class="Ruby">RUBY ON RAILS</li>
+                        <li class="Javascript">JAVASCRIPT</li>
+                    </ul>
+                    <button class="see-this-project">
+                        See this project
+                        <img src="images.svg\Union (7).svg" alt="" class="arrow">
+                    </button>
+                </div>
+            </div>
+            <div class="project-items">
+                <img src="images.svg\Rectangle 21 (19).svg" class="project-image" alt="">
+                <div class="languages">
+                    <h2>Project name goes here</h2>
+                    <ul class="H-R-J">
+                        <li class="Html-css">HTML/CSS</li>
+                        <li class="Ruby">RUBY ON RAILS</li>
+                        <li class="Javascript">JAVASCRIPT</li>
+                    </ul>
+                    <button class="see-this-project">
+                        See this project
+                        <img src="images.svg\Union (7).svg" alt="" class="arrow">
+                    </button>
+                </div>
+            </div>
+        </section>
+        <section class="section-3">
+            <h1 class="about-sign">About</h1>
+             <p>
+                Lorem ipsum, dolor sit amet consectetur adipisicing elit. Ab culpa iste nam qui excepturi facere at quis porro odio nulla in quod dignissimos, sint cumque quo! Repellat ipsum nulla a.
+             </p>
+             <button class="Start-collab">
+                Get my CV
+                <img src="images.svg\ic_download (2).svg" alt="">
+             </button>
+             <div class="color-boxes">
+                <div class="blue">
+                    <h3>Languages</h3>
+                    <ul>
+                        <li>Javascript(ES5/ES6)</li>
```

Comment:
There seems to be some horizontal scroll due to some elements overflowing. Kindly adjust this for a better user experience on the website. This will also help to centralize some elements that are not currently centralized.

![image](https://user-images.githubusercontent.com/20554583/205469480-9be3dd85-4094-4d2f-8343-e6e6f538cfe5.png)

Reply:
how do i do this please?